src/app/producer-shelf/components/producer-shelf/producer-shelf.component.html

```diff
@@ -0,0 +1,17 @@
+<div [style.flex-basis.px]="dragOffsetInPixels" [style.height.px]="dragOffsetInPixels">
+    <div #dragHandle class="drag-handle">
+        <sofie-icon-button [iconButton]="IconButton.BARS" [iconButtonSize]="IconButtonSize.XL"></sofie-icon-button>
+    </div>
+    <div
+        *ngFor="let keyBinding of keyBindings" (click)="keyBinding.action()"
+        class="keybinding"
+        style="padding: 12px 20px; margin: 10px;"
```

Comment:
This is inline styling. We don't want inline styling.

Reply:
What you could do is create some "helper" CSS classes that provides various forms of paddings and margins.
Something like `sp`, `mp`, `lp`, `spr`, `mpt`, `lm`.
The acronyms stands for `small padding`, `medium padding`, `large padding`, `small padding right`, `medium padding top` and `large margin`.

The acronums aren't pretty, but they are a handy way to quickly add paddings and margins to your markup.
You could discuss with Anders if this is something we want.

Reply:
I decided to delete the content in the `ProducerShelf`, since it is all being implemented in an entirely new fashion for @KvelaGorrrrnio's PR.

Reply:
> This is for development purposes,

We should try and not commit stuff for development purposes (we should definately not merge them). It's extremely easy to forget to remove them. Escpecially when it has been pushed and merged, because now there is no longer a git diff they will appear in, so you have to remember it yourself.

In my current draft PR I have committed some stuff for development purposes, but I intent to remove them before marking the PR ready for review.